bundles/net.tourbook/src/net/tourbook/database/TourDatabase.java

```diff
@@ -7317,6 +7317,16 @@ private int updateDbDesign_039_to_040(final Connection conn, final SplashManager
             // Create index in table: TOURDATA_TOURTAG - Index: TOURTAG_TAGID
             SQL.CreateIndex(  stmt, JOINTABLE__TOURDATA__TOURTAG, KEY_TAG);
 
+// SET_FORMATTING_ON
+         }
+
+         if (isColumnAvailable(conn, TABLE_TOUR_COMPARED, "tourRecordingTime") == false) { //$NON-NLS-1$
+
+// SET_FORMATTING_OFF
+
+            // Add new columns
+            SQL.AddCol_Float(stmt, TABLE_TOUR_COMPARED, "tourRecordingTime", DEFAULT_0); //$NON-NLS-1$
+
```

Comment:
There is already one column checked for this version, so I think it is not necessary to check a second column

Reply:
Right, I will remove

Reply:
Fixed, see my [PR](https://github.com/wolfgang-ch/mytourbook/pull/153/)